jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/Jar.java

```diff
@@ -44,6 +44,14 @@
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private Path jarFile;
 
+  @CommandLine.Option(
+      names = "--mode",
+      defaultValue = "exploded",
+      paramLabel = "<mode>",
+      description = "The jar processing mode, candidates: packaged, default: exploded")
```

Comment:
Why not creating a dedicated type and have the descriptions like other options?
```
"set console output type, candidates: ${COMPLETION-CANDIDATES}, default: ${DEFAULT-VALUE}")
```
I think it will take care of validation for us too.